cmd/request-bottle.rb

```diff
@@ -55,7 +57,11 @@ def request_bottle
     odie "Email not specified" if email.empty?
 
     Homebrew.args.resolved_formulae.each do |formula|
-      payload = { formula: formula.name, name: user, email: email, ignore_errors: Homebrew.args.ignore_errors? }
+      payload = { formula:       formula.name,
+                  name:          user,
+                  email:         email,
+                  ignore_errors: Homebrew.args.ignore_errors?,
+                  issue:         Homebrew.args.issue }
```

Comment:
```suggestion
                  issue:         Homebrew.args.issue || 0 }
```

By looking at that line in this commit: https://github.com/Homebrew/linuxbrew-core/commit/06c199d8af10533da011e2901751c2bdcace7a14#diff-ed1972b16778b95d046bd90b3bd93647R64

Am I thinking correctly?

Reply:
I was relying on Javascript type coercion but it's probably better to be more explicit.